src/motor_control/src/motor_control_node.cpp

```diff
@@ -196,6 +204,10 @@ class MotorControlNode : public rclcpp::Node {
     if (this->pid_controllers[id]) {
       this->pid_controllers[id]->setRotation(position);
     }
+    while(abs(this->can_data[id].tachometer - this->pid_controllers[id]->getTargTach()) > this->tachThreshold)
+    {
+      continue;
+    }

```

Comment:
What do you do if you want to stop the motor before it reaches its final position